test/OptionSettlementEngine.t.sol

```diff
@@ -162,6 +162,17 @@ contract OptionSettlementTest is Test, NFTreceiver {
         assertTrue(!claim.claimed);
     }
 
+    function testWriteClaimWithOptionId() public {
```

Comment:
We can better express the issue by using another actor who tries to write more options to an existing claim. Might be as simple as pranking the second call to write from Bob.

In other words, is there a test case where, without this check, we could have avoided hitting the `CallerDoesNotOwnClaimId` error even when we don't own the claim ID?